src/test/java/mercadopago/BaseResourceTest.java

```diff
@@ -0,0 +1,24 @@
+package mercadopago;
+
+import com.mercadopago.MercadoPagoConfig;
+import com.mercadopago.net.MPDefaultHttpClient;
+import java.util.TimeZone;
+import mercadopago.mock.HttpClientMock;
+import mercadopago.mock.MPDefaultHttpClientMock;
+import org.apache.http.client.HttpClient;
+import org.junit.jupiter.api.BeforeAll;
+
+/** BaseResourceTest class. */
+public abstract class BaseResourceTest extends MPDefaultHttpClient {
```

Comment:
```suggestion
public abstract class BaseClientTest extends MPDefaultHttpClient {
```
Uma sugestão, mesmo nome usado em dotnet.
Por que extender `MPDefaultHttpClient`?

Reply:
Realmente não tem sentido, eu havia extendido quando estava tentando uma outra coisa e esqueci de retirar. Alterações realizadas.